### PR TITLE
Fix edit links in pay and owe

### LINKED ISSUE #264: Fix links to edit command in owe and pay

### PR DESCRIPTION
**Pull request by Soumyadip-cmd**
Closes #264

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -291,7 +291,7 @@ Updates the amount of tuition fee owed by a specified student after a lesson.
 
 <box type="tip" header="##### Tips">
 
-* In case you accidentally make a mistake using the <md>`owe`</md> command, you can use the [`edit` command](#editing-a-student--edit) to fix the OWE_AMOUNT as your preference.
+* In case you accidentally make a mistake using the <md>`owe`</md> command, you can use the [`edit` command](#editing-a-student-edit) to fix the OWE_AMOUNT as your preference.
```
Comment (teamotitan) [inline docs/UserGuide.md:294]: do you think its made instead of make?

**Review by teamotitan — COMMENTED** [review]
Impressive sir, one small comment

File: docs/UserGuide.md [hunk 2]
```diff
@@ -291,7 +291,7 @@ Updates the amount of tuition fee owed by a specified student after a lesson.
 
 <box type="tip" header="##### Tips">
 
-* In case you accidentally make a mistake using the <md>`owe`</md> command, you can use the [`edit` command](#editing-a-student--edit) to fix the OWE_AMOUNT as your preference.
+* In case you made a mistake using the <md>`owe`</md> command, you can use the [`edit` command](#editing-a-student-edit) to fix the OWE_AMOUNT to your preference.
```
Comment (DennieDan) [inline docs/UserGuide.md:294]: Thanks for spotting this mistake

**Review by DennieDan — APPROVED** [review]

**Review by TY1Fan — APPROVED** [review]
LGTM

**Pull request merged**